frontend/mr-admin-flate/src/components/avtaler/OpprettAvtaleContainer.tsx

```diff
@@ -401,6 +402,7 @@ export function OpprettAvtaleContainer({
                     label: "Sluttdato",
                   }}
                 />
+                {redigeringsModus ? <AvbrytAvtale onAvbryt={onClose} /> : null}
```

Comment:
Her kan du også skrive `{redigeringsModus && <AvbrytAvtale onAvbryt={onClose} />}`, synes hvertfall det blir noe penere kode?

Reply:
Det kan jeg, men det kan føre til bugs. Her er en artikkel fra en velkjent React-guru om bruk av `&&` vs ternary https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Stemmer det, denne viste du meg tidligere!